zebrad/src/components/sync/tests/vectors.rs

```diff
@@ -0,0 +1,134 @@
+use futures::future;
+use std::sync::{
+    atomic::{AtomicU8, Ordering},
+    Arc,
+};
+use tokio::{
+    runtime::Runtime,
+    time::{timeout, Duration},
+};
+
+use super::super::*;
+use crate::config::ZebradConfig;
+
+/// Make sure the timeout values are consistent with each other.
+#[test]
+fn ensure_timeouts_consistent() {
```

Comment:
Please add a timeout consistency test for `GENESIS_TIMEOUT_RETRY`.

It should be longer than the handshake timeout in zebra_network.

Do you think there should be a maximum as well? Maybe the block download timeout?

Reply:
https://github.com/ZcashFoundation/zebra/pull/2255/commits/85281721245bc142a7da27a12ffb84efc795f213